src/Mod/Draft/DraftTrackers.py

```diff
@@ -741,6 +741,20 @@ def get(self):
         p = self.coords.point.getValues()[0]
         return Vector(p[0],p[1],p[2])
 
+    def getDocName(self):
+        return str(self.selnode.documentName.getValue())
+
+    def getObjName(self):
+        return str(self.selnode.objectName.getValue())
+
+    def getSubelementName(self):
+        return str(self.selnode.subElementName.getValue())
+        
+    def getSubelementIndex(self):
+        subElement = self.getSubelementName()
+        idx = int(subElement[8:])
+        return idx
+
```

Comment:
I suggest using `snake_case` for these methods. I know most of Draft doesn't use this style, but we should try to change it, little by little. As long as we don't break backwards compatibility, of course.

Reply:
ok, I can change all Draft Edit to snake_case without any problems for backward compatibility. Anyway there will alwais i think something in the whole Draft WB that we wont be able to change, ad it will look wierd... isnt'it better to have all camelCase?

Reply:
No. Because those CamelCase functions are provided by C++ classes, so we will know they come from C++. But the rest we can be confident is defined in Python.